incident-management/pagerduty.yaml

```diff
@@ -0,0 +1,121 @@
+---
+# Description
+#   The Sensu PagerDuty Handler is a Sensu Event Handler that alerts to
+#   the PagerDuty incident reponse service.
+#
+# Instructions
+#   1. Add "pagerduty" to the "incident-management" handler set.
+#
+#      To create the "incident-management" handler set, please run:
+#
+#      $ sensuctl handler create incident-management --type set --handlers pagerduty
+#
+#      If you already have a handler set called "incident-management", please run:
+#
+#      $ sensuctl edit handler incident-management
+#
+#      NOTE: the sensuctl edit command will open your default editor; please
+#      modify the "handlers" attribute by adding "pagerduty".
+#
+#   2. Configure the following secrets using your preferred Sensu Go Secrets
+#      provider:
+#
+#      - pagerduty_token
+#
+#      NOTE: example secret resources using the "env" provider are included in
+#      this template; to add your secrets to the Sensu Backend, please modify
+#      /etc/default/sensu-backend or /etc/sysconfig/sensu-backend and restart
+#      the sensu-backend service.
+#
+#   3. More configuration options are available for this plugin; please see the
+#      plugin documentation for more details.
+#
+# Documentation
+#   - https://github.com/sensu/sensu-pagerduty-handler#sensu-pagerduty-handler
+#   - https://docs.sensu.io/sensu-go/latest/reference/secrets/
+#
+# Contributors
+#   The following individuals have contributed to this configuration template:
+#   - Todd Campbell, @nixwiz
+type: Handler
+api_version: core/v2
+metadata:
+  name: pagerduty
+spec:
+  command: sensu-pagerduty-handler
+  env_vars: null
+  filters:
+  - is_incident
+  - not_silenced
+  handlers: null
+  runtime_assets:
+  - sensu/sensu-pagerduty-handler:1.3.2
+  secrets:
+  - name: PAGERDUTY_TOKEN
+    secret: pagerduty_token
+  timeout: 0
+  type: pipe
+---
+type: Secret
+api_version: secrets/v1
+metadata:
+  name: pagerduty_token
+spec:
+  provider: env
+  id: PAGERDUTY_TOKEN
+---
+type: Asset
+api_version: core/v2
+metadata:
+  name: sensu-pagerduty-handler:1.3.2
+  labels: 
```

Comment:
Should we drop the labels key since it's not being used?

Reply:
Yeah, I copy/pasted that right from Bonsai.  I'll take it out.